contrib/bind/provider.py

```diff
@@ -43,8 +43,8 @@ def remove_record(self, record):
 
     def first_setup(self, parser):
         # Insert SOA and NS records
-        hostname = unit_get('public-address')
-        addr = resolve_hostname_to_ip(hostname)
+        addr = unit_get('public-address')
+        #addr = resolve_hostname_to_ip(hostname)
```

Comment:
unit_get is now returning the IP and not the name anymore.. 
resolve_hostname fails when giving an IP instead of a name :(..